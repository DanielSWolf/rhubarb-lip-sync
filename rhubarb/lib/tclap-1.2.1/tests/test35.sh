#!/bin/sh

# failure
../examples/test6 -n mike 2  > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test35.out; then
	exit 0
else
	exit 1
fi

