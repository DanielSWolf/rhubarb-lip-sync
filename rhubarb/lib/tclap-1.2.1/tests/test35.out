PARSE ERROR: Argument: -n (--name)
             Value 'mike' does not meet constraint: homer|marge|bart|lisa|maggie

Brief USAGE: 
   ../examples/test6  -n <homer|marge|bart|lisa|maggie> [--] [--version]
                      [-h] <1|2|3>

For complete USAGE and HELP type: 
   ../examples/test6 --help

